.github/workflows/audit.yaml

```diff
@@ -25,7 +25,8 @@ jobs:
         python-version: ${{inputs.python_version}}
     - name: Install Python packages
       run: |
-        pip install --upgrade pip six setuptools pytest codecov 'coverage[toml]<=6.2'
+        pip install --upgrade pip
+        pip install '.[dev,ci]' # not using -e because of python 2.7 support
```

Comment:
coverage output is readable by default without rewrite config

Reply:
Yeah, really it's to make sure that we don't accidentally run spack from the install location, for reasons like @ofek mentions.  We point to the in-tree version anyway, so it should be fine, but it's one less source of confusion.  If there were a way to only install the dependencies rather than the dependencies *and spack* that's what I would want to use, but it seems like that isn't a supported use-case in pip.